src/res.c

```diff
@@ -315,7 +315,7 @@ bool loadMedia() {
   } else if (!loadUnifont()) {
     fputs("Failed to load unifont! SDL_ttf Error: ", stderr);
     fputs(TTF_GetError(), stderr);
-    fputchar('\n', stderr);
+    fputs('\n', stderr);
```

Comment:
This is simply wrong and may cause a segfault. '\n' is a char, not a pointer.

Reply:
I don't see why this change is needed.

Reply:
OK, I fixed it.